db/seeds/create_users.rb

```diff
@@ -1,13 +1,13 @@
 unless User.exists?(email: "dev@hfeyp.com")
   User.create! do |u|
     u.email = "dev@hfeyp.com"
-    u.password = "Partyw0rd"
-    u.role = "editor"
+    u.password = "11abXY!@"
+    u.role = "admin"
   end
 
   User.create! do |u|
     u.email = "guy@home.com"
-    u.password = "guyguy"
+    u.password = "11abXY!@"
     u.role = "editor"
```

Comment:
We need to avoid seeding public servers with passwords that are in the repo

Reply:
agreed, but what's the alternative?